plugins/kubernetes/config/routes.rb

```diff
@@ -32,6 +32,10 @@
       end
     end
     resources :usage_limits, except: [:edit]
-    resources :namespaces, except: [:edit]
+    resources :namespaces, except: [:edit] do
+      collection do
+        post :sync_all
```

Comment:
also able to sync a single resource?

Reply:
added a `sync` button to the edit view too